content/08-mongodb/09-mongodb-indexes.mdx

```diff
@@ -0,0 +1,260 @@
+        isUnique: false,
+        isSparse: false,
+        isPartial: false,
+        indexVersion: 2,
+        direction: 'forward',
+        indexBounds: { name: [ '["Ramsay Bolton", "Ramsay Bolton"]' ] }
+      }
+    },
+    rejectedPlans: []
+  },
+  executionStats: {
+    executionSuccess: true,
+    nReturned: 273,
+    executionTimeMillis: 0,
+    totalKeysExamined: 273,
+    totalDocsExamined: 273,
+    executionStages: {
+      stage: 'FETCH',
+      nReturned: 273,
+      executionTimeMillisEstimate: 0,
+      works: 274,
+      advanced: 273,
+      needTime: 0,
+      needYield: 0,
+      saveState: 0,
+      restoreState: 0,
+      isEOF: 1,
+      docsExamined: 273,
+      alreadyHasObj: 0,
+      inputStage: {
+        stage: 'IXSCAN',
+        nReturned: 273,
+        executionTimeMillisEstimate: 0,
+        works: 274,
+        advanced: 273,
+        needTime: 0,
+        needYield: 0,
+        saveState: 0,
+        restoreState: 0,
+        isEOF: 1,
+        keyPattern: { name: 1 },
+        indexName: 'name_1',
+        isMultiKey: false,
+        multiKeyPaths: { name: [] },
+        isUnique: false,
+        isSparse: false,
+        isPartial: false,
+        indexVersion: 2,
+        direction: 'forward',
+        indexBounds: { name: [ '["Ramsay Bolton", "Ramsay Bolton"]' ] },
+        keysExamined: 273,
+        seeks: 1,
+        dupsTested: 0,
+        dupsDropped: 0
+      }
+    }
+  }
+}
+```
+Comparing the same fields as the un-indexed query we see that the `stage` from the `winningPlan` is now `IXSCAN` which indicates an index was used.
+
+Additionally, we see that `totalDocsExamined` is now just the 273 documents where `name` is `"Ramsay Bolton"` and not the entire 50,303 documents. This increased efficiency can be see especially now with `executionTimeMillis` totaling 0ms. 
+
+Analyzing the explain plan on your most important queries will show you the performance of your index or if one needs to be created to increase the efficiency of your application. 
+
+## How to drop an index
+While the explain plan may show the need for an index, it can also do the opposite. For example, if an index is no longer needed or not adding that much improved performance, it may be in your best interest to drop that index and reserve the space or preserve some write performance. 
+
+To drop an index on your collection, the basic syntax is as follows using the `dropIndexes()` method:
+
+```
+db.COLLECTION_NAME.dropIndex( { "FIELD_NAME": 1 } )
+```
+If we wanted to drop our `country` index example from earlier, we would write the following:
+
+```
+db.mycoll.dropIndex( { "country":1 } )
+```
+
+## Conclusion
+Efficiently querying your database leads to an improved user experience of your application. Additionally those working with the data for analysis or other internal efforts will have faster performance and ease of working with the database. Knowing how to index and how indexes work is key to achieving query efficiency. 
```

Comment:
```suggestion
Efficiently querying your database leads to an improved user experience of your application. Additionally those using with the data for analytics or other internal efforts will have faster performance and ease of working with the database. Knowing how to index and how indexes work is key to achieving query efficiency. 
```

I changed "working" -> "using" here just to mix it up from the "working" later in the sentence.